app/power.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <stdbool.h>
+#include "errors.h"
+
+#define EPOWER_NO_INIT			(POWER_BASE + 0)
+#define EPOWER_TASK_CREATE		(POWER_BASE + 1)
+#define EPOWER_QUEUE_CREATE		(POWER_BASE + 2)
+#define EPOWER_VOLTAGE_RANGE	(POWER_BASE + 3)
+
+void power_dut_set_enabled(bool enabled);
+err_t power_dut_ldo_set(uint32_t millivolt);
```

Comment:
Maybe add getters for these two as well, for future ;)

Reply:
Sure